dataactbroker/handlers/fileHandler.py

```diff
@@ -1176,7 +1176,7 @@ def get_error_metrics(submission):
         return JsonResponse.error(e, StatusCode.INTERNAL_ERROR)
 
 
-def list_submissions(page, limit, certified):
+def list_submissions(page, limit, certified, sort='modified', order='desc'):
```

Comment:
actually, if i remove the default values here then the py.test will fail because it calls the function directly